fragments/tap-initialize/config/workload.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: my-project
   labels:
     apps.tanzu.vmware.com/workload-type: web
+    apps.tanzu.vmware.com/has-tests: "true"
```

Comment:
Nice to see this in the fragment.

Reply:
actually, not sure it should always be there, it is added based on an option here: https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/fragments/tap-initialize/accelerator.yaml#L41-L45

Reply:
removed it